### ISSUE TITLE
Verified domain popup has incorrect link hover style

### ISSUE THREAD

**Issue by xt0rted** [StylishThemes/GitHub-Dark#729]
Part of #728 

* **Browser**: chrome
* **Operating System**: win10
* **Link**: https://github.com/airbnb
* **Screenshot**:

![image](https://user-images.githubusercontent.com/831974/43841897-28e0d554-9af2-11e8-91a0-e0ae25b4177a.png)

* **HTML of the section where the issue occurs**:

````html
<div class="Popover dropdown-menu-content anim-scale-in js-menu-content">
  <div class="Popover-message Popover-message--large text-left p-3 mt-2 Box box-shadow-large dropdown-menu dropdown-menu-s py-0 text-gray-dark text-left">
    <p>
      We've verified that the organization <strong>Airbnb</strong> controls the domain:
    </p>
    <ul>
        <li class="mb-1">
          <svg class="octicon octicon-verified mr-1 text-green" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M15.67 7.066l-1.08-1.34a1.5 1.5 0 0 1-.309-.77l-.19-1.698a1.51 1.51 0 0 0-1.329-1.33l-1.699-.19c-.3-.03-.56-.159-.78-.329L8.945.33a1.504 1.504 0 0 0-1.878 0l-1.34 1.08a1.5 1.5 0 0 1-.77.31l-1.698.19c-.7.08-1.25.63-1.33 1.329l-.19 1.699c-.03.3-.159.56-.329.78L.33 7.055a1.504 1.504 0 0 0 0 1.878l1.08 1.34c.17.22.28.48.31.77l.19 1.698c.08.7.63 1.25 1.329 1.33l1.699.19c.3.03.56.159.78.329l1.339 1.08c.55.439 1.329.439 1.878 0l1.34-1.08c.22-.17.48-.28.77-.31l1.698-.19c.7-.08 1.25-.63 1.33-1.329l.19-1.699c.03-.3.159-.56.329-.78l1.08-1.339a1.504 1.504 0 0 0 0-1.878zm-9.164 4.936L3.008 8.505l1.5-1.5 1.998 2 4.997-4.997 1.499 1.549-6.496 6.445z"></path></svg>
          <strong>airbnb.io</strong>
        </li>
    </ul>
    <div class="pt-2">
      <a href="https://help.github.com/articles/verifying-your-organization-s-domain/">Learn more about verified organizations</a>
    </div>
  </div>
</div>
````

**Comment by xt0rted** [comment]
The issue is coming from the `.dropdown-menu a:hover` rule

**Comment by the-j0k3r** [comment]
Added a higher selector range https://github.com/StylishThemes/GitHub-Dark/commit/4a6c360d01ca02980b5088775ec62f87ecb9e0c2 in order to fix #732 and maybe not break this incidentally..
